lib/srv/db/mongodb/engine.go

```diff
@@ -206,7 +207,7 @@ func (e *Engine) processHandshakeResponse(ctx context.Context, respMessage proto
 	case *protocol.MessageOpMsg:
 		rawMessage = bson.Raw(resp.BodySection.Document)
 	default:
-		e.Log.Warn("Unabled to process MongoDB handshake response. Unexpected message type %T", respMessage)
+		e.Log.WarnContext(ctx, "Unable to process MongoDB handshake response. Unexpected message type.", "message_type", fmt.Sprintf("%T", respMessage))
```

Comment:
```suggestion
		e.Log.WarnContext(ctx, "Unable to process MongoDB handshake response. Unexpected message type.", "message_type", log.TypeAttr(respMessage))
```

log is lib/utils/log